README.md

```diff
@@ -41,8 +41,15 @@ const { PACKETS } = constants;
 *   'port' is optional, defaults to 20777
 *   'bigintEnabled' is optional, setting it to false makes the parser skip bigint values,
 *                   defaults to true
+*   'parserEnabled' is optional, setting it to false makes the client skip the parsing,
+*                   which can be a bottleneck on high frequencies. you can then parse
+*                   using the exposed parsing functions from
+*                   defaults to true
+*   'forwardPort' is optional, passing it a number will make the client bridge the messages
+*                   over to that port. this is useful to send telemetry to connected peripherals.
+*                   defaults to undefined
 */
-const client = new F1TelemetryClient({ port: 20777, bigintEnabled: true });
+const client = new F1TelemetryClient({ port: 20777, forwardPort: 4477, bigintEnabled: true, bridgeMode: true });
```

Comment:
most peripherals use 30500 as the port, so having the `forwardPort` default to 30500 would be good